pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -601,13 +649,28 @@ func (r *ReconcileHyperConverged) ensureNetworkAddons(instance *hcov1alpha1.Hype
 
 	found := &networkaddonsv1alpha1.NetworkAddonsConfig{}
 	err = r.client.Get(context.TODO(), key, found)
+
 	if err != nil && apierrors.IsNotFound(err) {
 		logger.Info("Creating Network Addons")
 		return r.client.Create(context.TODO(), networkAddons)
 	} else if err != nil {
 		return err
 	}
 
+	existingOwners := found.GetOwnerReferences()
+
+	// Previous versions used to have HCO-operator (scope namespace)
+	// as the owner of NetworkAddons (scope cluster).
+	// It's not legal, so remove that.
+	if (len(existingOwners) > 0) {
+		logger.Info("NetworkAddons has owners, removing...")
```

Comment:
consider clearer log message for example: Removing NetworkAddonsConfigs owner references